### PR TITLE
PLATOPS-1566 Migration to the New Build; added Play cross compilation

### PR DESCRIPTION
**Pull request by jachro**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+libraryDependencies ++= LibDependencies()
+
+crossScalaVersions := Seq("2.11.8")
+
+resolvers := Seq(
+  Resolver.bintrayRepo("hmrc", "releases"),
+  "typesafe-releases" at "http://repo.typesafe.com/typesafe/releases/"
+)
+
+collaborators
```
Comment (konradwudkowski) [inline build.sbt]: Maybe we could remove collaborators (don't quite see a reason for it) and we could make this file look like all other `build.sbt` i.e. val root = Project(name, ".").settings(...) rather then having all settings at root level. Just to keep it consistent with other services/libs

File: project/LibDependencies.scala [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+  def apply(): Seq[ModuleID] = compile ++ test
+
+  private val play25Version = "2.5.12"
+  private val play26Version = "2.6.15"
+
+  private val compile: Seq[ModuleID] = PlayCrossCompilation.dependencies(
+    shared = Seq(
+      "com.codahale.metrics" % "metrics-graphite" % "3.0.2",
+      "de.threedimensions"   %% "metrics-play"    % "2.5.13"
```
Comment (konradwudkowski) [inline project/LibDependencies.scala]: I remember using a different metrics reporting library for play 2.6

**Review by konradwudkowski — CHANGES_REQUESTED** [review]

File: build.sbt (hunk 1 above)
Comment (jachro) [reply build.sbt]: The `collaborators` are removed.

In terms of the build.sbt structure I didn't change it as other Digital Contact libraries follow that pattern.

File: project/LibDependencies.scala (hunk 2 above)
Comment (jachro) [reply project/LibDependencies.scala]: I aligned these dependencies so they are the same as in both bootstrap-play-25 and bootstrap-play-26.

**Pull request merged**